senders/telegram/send.go

```diff
@@ -3,7 +3,6 @@ package telegram
 import (
 	"bytes"
 	"fmt"
-
```

Comment:
We sort imports with `goimports`, so this empty line should stay.

We probably need to set up a linter for this @litleleprikon.

Reply:
Or just `desc`, since it's equal to `trigger.Desc`.

Reply:
We have a problems only with one linter in golangci-lint. This linter is `golint` as we have a bunch of unchecked returned errors. I believe that we can setup more strict linter. I will check it